README.md

```diff
@@ -18,11 +18,20 @@ IntelliSense is natively supported for JavaScript, TypeScript, JSON, HTML, CSS,
 - ⌃⌘G: View: Show SCM
 - ⇧⌘D: View: Show Run and Debug
 - ⇧⌘X: View: Show Extensions
+- ⌘1, ⌘2, ...: View: Focus First, Second, ... Editor Group
+- ⌃1, ⌃2, ...: View: Focus on First, second, ... Editor in Group
```

Comment:
Damn! Didn't even know about these but I'll use them from now on. Two things:

1. Could we put the intra-editor-group navigation commands before the inter-editor-group ones?
2. Can we add relative editor group navigation commands as well in addition to the absolute ones you have here?

Again, I'm doing this more so we can test out the GitHub code review pipeline. Thanks again!

Reply:
1. [todo] argument against thing 1
2. no, we cant